evap/evaluation/models.py

```diff
@@ -624,6 +632,10 @@ def is_in_evaluation_period(self):
     def general_contribution_has_questionnaires(self):
         return self.general_contribution and self.general_contribution.questionnaires.count() > 0
 
+    @property
+    def has_valid_main_language(self):
```

Comment:
How do you feel about this, so that we don't add any more vocabulary?
```suggestion
    def has_decided_main_language(self):
```